src/routes/applications.ts

```diff
@@ -258,27 +258,29 @@ const createApplicationsRouter = (
       const csv = await createDacoCSVFile(req);
       // encrypt csv content, return {content, iv}
       const config = await getAppConfig();
-      const encrypted = await encrypt(csv, config.auth.DACO_ENCRYPTION_KEY);
-
-      if (encrypted?.content) {
```

Comment:
this if  is still good
```
if(!encrypted) {
 thorw new Error('failed to encrypt');
}

and the catch would catch it and return 500 error response
```

Reply:
ok I see that below this function throws an error

Reply:
the problem is the return type says it can be undefined but that's not correct since it's either an object or exception if it fails